Remove CI image (#370)
3scale/apisonator: DEVELOPMENT.md

Next cd into the directory, `cd apisonator`.

### Services

Apisonator requires a Redis server to run. By default, it will try to connect to:

```
redis://localhost:6379
```

A [podman-compose.yml](script/config/podman-compose.yml) file is provided in order
to make it easier to launch some preconfigured setups, like sentinels or twemproxy.

To launch a single redis instance listening in the default URL, just run:

```
podman-compose -f script/config/podman-compose.yml start redis-master
```

Keep in mind that the test suite can't be launched against a twemproxy.

### Configuring tests environment

The test suite will read the environment variables set in `/.env.test`.

You can set any [documented](docs/configuration.md) variable, for example, to make
the test suite point to another redis instance.

## Running tests

> To execute all tests from within the development environment
> is to execute them via this script:

Reviewer: ```suggestion
To execute all tests from within the development environment, run this script:
```

Author: I'll fix this... 🤕

> `$ bundle exec script/test`

### Running tests (advanced)

> TYou can execute the following commands to execute all tests:

Reviewer: ```suggestion
You can execute the following commands to run all tests:
```

just to avoid double execute :)

Author: 👍

## Running tests

To execute all tests from within the development environment, run this command:

> > `$ bundle exec script/test`º

Reviewer: :eyes: `º`

Author: I dºn't see the problem

Author: Fixed

1. Change the contents of version.rb with the new version.
2. Run `bundle install` with all Gemfiles to update all lockfiles.
3. Run `rake license_finder:report:xml > licenses.xml`.
4. Modify CHANGELOG.md filling up data about notable changes along with
   associated issue or PR numbers where available.

> 5. Run task release:changelog:link_prs to link the issue or PR numbers in the

Reviewer: ```suggestion
5. Run `bundle exec rake release:changelog:link_prs` to link the issue or PR numbers in the
```

Author: 👍

   CHANGELOG.md file and check the diff makes sense.
6. Review and commit "apisonator: release X.Y.Z".
7. Verify the tests pass and fix any issue that crops up. If at all possible,
   deploy the version as a pre-release in a staging environment for further
   testing and early detection of issues.

> 8. If you did step 4, build the new CI image, rebuild the dev image based on it,

Reviewer: Is this step still needed?

Author: Not really. And I'm not sure what to do with the steps above mentioning the CHANGELOG
